src/common/state/ColorControlPointList.xml

```diff
@@ -59,4 +65,10 @@
     </Function>
     <Function name="ProcessOldVersions" user="true" member="true">
     </Function>
+    <Include file="source" quoted="false">
+      DebugStream.h
+    </Include>
+    <Include file="header" quoted="false">
+      algorithm
```

Comment:
Why is algorithm needed in the header ?

Reply:
Is there a better way to include it? I think it was needed for `std::all_of`. Should I include it instead with the function where I need it in the `.code` file?

Reply:
change 'header' to 'source', because it isn't needed in the header, correct?

Reply:
ahh that would make sense. Thanks!